### PR TITLE
fix block number to prevent failure on `get_logs` tests

### PR DESCRIPTION
**Pull request by tcoratger**
Time spent on this PR:

Resolves: #1097

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?

# Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: tests/tests/eth_provider.rs [hunk 1]
```diff
@@ -20,6 +20,7 @@ use reth_primitives::{
 };
```
Comment (greged93) [inline tests/tests/eth_provider.rs:20]: Is there no way to fix this from within the mongo fuzzer structure? For example by adding a `current_block` field to the fuzzer and using this to set the block number of logs?

Comment (tcoratger) [reply tests/tests/eth_provider.rs:20]: Just changed that directly inside the Mongo Fuzzer. Let us check with the CI if it doesn't break anything else in the tests.

File: tests/tests/eth_provider.rs [hunk 2]
```diff
@@ -20,6 +20,7 @@ use reth_primitives::{
...
 use reth_rpc_types::request::TransactionInput;
 use reth_rpc_types::serde_helpers::JsonStorageKey;
+use reth_rpc_types::Header;
```
Comment (greged93) [inline tests/tests/eth_provider.rs:23]: is this needed?

Comment (tcoratger) [reply tests/tests/eth_provider.rs]: Fixed

**Review by greged93 — DISMISSED** [review]
🤞 on CI

**Pull request merged**